PROJECT

```diff
@@ -57,4 +57,7 @@ resources:
   kind: VolumeReplicationClass
```

Comment:
did you use a `kubebuilder` command to generate the initial files? Please include the command(s) in the commit message so that others can reproduce it, or follow the steps to add other webhooks.